sql/Cargo.toml

```diff
@@ -27,9 +27,12 @@ datafusion-expr = { workspace = true }
 datafusion-proto = { workspace = true }
 df_operator = { workspace = true }
 hashbrown = { version = "0.12", features = ["raw"] }
+influxdb_influxql_parser = { git = "https://github.com/Rachelint/influxdb_iox.git", branch = "influxql-parser" }
```

Comment:
What changes are in this branch?

Reply:
No 

> > User can use some simple raw queries in influxql to query data in CeresDB.
> 
> Give some example which kinds of QL are supported by this PR.

Ok, it is planned to add in later.

Reply:
In fact, no actual changes, just reduce the dependencies.